lib/editor/components/CreateSnapshotModal.js

```diff
@@ -112,13 +145,36 @@ class CreateSnapshotModal extends Component<Props, State> {
               {this.messages('fields.publishNewVersion.label')}
             </Checkbox>
           </FormGroup>
+          {loading && <Icon type='spinner' className='fa-pulse' />}
+          {unapprovedRoutes.length > 0 &&
+            <div>
+              <h4>{this.messages('unapprovedRoutesHeader')}</h4>
+              <h5>{this.messages('unapprovedRoutesDesc')}</h5>
+              <ul>
+                {
+                  unapprovedRoutes.map(route => (
+                    <li key={route.route_id}>{getEntityName(route)}: {getRouteStatusText(parseInt(route.status))}</li>
```

Comment:
Glad we moved to `getEntityName`, but I think we should include the route longName if possible! I think there's a helper method somewhere. As it stands the following can happen (if there's 2 routes with the same shortName)
<img width="401" alt="Screen Shot 2021-12-06 at 2 10 04 PM" src="https://user-images.githubusercontent.com/86619099/144851775-8b7c9063-e43c-4ac9-8dab-70df8f036db7.png">

Reply:
getEntityName *should* use the routeLongName... but it doesn't use routeDesc, which some agencies use in lieu of the long name. 

https://github.com/ibi-group/datatools-ui/issues/745

Reply:
In your screenshot, do those routes have long names or descriptions? If they are long names, they should be shown.

Reply:
I think this should be fixed in a separate PR.

Reply:
I agree! Thanks for opening #745. Will resolve this for now.